### PR TITLE
Add basic quality checks

### PR DESCRIPTION
**Pull request by kevin-hausmann**

### CODE REVIEW

**Review by Christian-Mielke — APPROVED** [review]

File: Convert ePRTR data to CLRTAP LPS.ipynb [hunk 1]
```diff
@@ -110,13 +129,158 @@
...
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Check total emissions\n",
+    "sum = DataFrame({\"Sum LPS\": data.iloc[:, 8:23].sum()})\n",
+    "sum[\"National Total\"] = [1210.50, 1125.02, 291.78, 601.24, 94.66, 207.01, 2957.80, 162.18, 11.94, 8.30, 119.8, 75.71, 12.82, 219.33]\n",
```
Comment (AnnaZagorski) [inline Convert ePRTR data to CLRTAP LPS.ipynb:162]: I got different totals for the heavy metals

File: Convert ePRTR data to CLRTAP LPS.ipynb [hunk 2]
```diff
@@ -125,7 +289,6 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# Optional: Create the shapefile and inline map to visualize data\n",
     "import geopandas\n",
```
Comment (AnnaZagorski) [inline Convert ePRTR data to CLRTAP LPS.ipynb:292]: this still does not work for me

**Review by AnnaZagorski — APPROVED** [review]
I got different totals for the heavy metals

File: Convert ePRTR data to CLRTAP LPS.ipynb (hunk 2 above)
Comment (AnnaZagorski) [inline Convert ePRTR data to CLRTAP LPS.ipynb:292]: this still does not work for me

**Review by AnnaZagorski — APPROVED** [review]

**Pull request merged**